app/src/main/res/values/styles.xml

```diff
@@ -465,11 +465,14 @@
     <item name="android:gravity">center_horizontal</item>
   </style>
 
-  <!-- ShapeableImageView
-     Note that When defining a style to be used as a shapeAppearanceOverlay in shapeableImageView , we generally
-     set an empty parent by setting parent="". This is what causes the overlay behavior and also
-     shapeAppearanceOverlay stacks on top of shapeAppearance, so all values will be set in the shapeAppearance.
-     context for ShapeableImageView https://howtodoandroid.com/shapeableimageview-material-components-android/ -->
+  <!-- ShapeableImageView -->
+  <!-- Note that when defining a style to be used as a shapeAppearanceOverlay in shapeableImageView , we generally
+       set an empty parent by setting parent="".This is what causes the overlay behavior.
+       When we define a style,we would provide a parent style, and then we can override specific attributes to alter specific parts of that style.
+       When we specify no parent, then the style will overlay whatever theme is being applied. Applying explicit parent attribute instead of a empty
+       parent may cause overriding(other properties may get added i.e. color or theme used in activity).
+       Thus here in ShapeAppearanceOverlay we have defined a empty parent to allow the active theme behavior to apply.
+       https://blog.stylingandroid.com/material-shape-shapeappearance/ -->
```

Comment:
So after re-reading the Blog, I'm starting to wonder if we're actually taking the correct approach. It seems setting ``parent=""`` only incidentally fixes the problem since the theme just happens to have the correct default corner family type. Could we perhaps not specify an empty parent and instead specify a corner family of rounded?

Per https://material.io/develop/android/theming/shape, https://medium.com/androiddevelopers/material-theming-with-mdc-shape-126c4e5cd7b4, and https://material.io/design/shape/about-shape.html#shape-customization-tool it seems like that should work. It'll also be easier to document since we can just say something like:

```
The corner size & family type are set up to ensure a circular shape is created for components with equal width & height (per https://material.io/design/shape/about-shape.html#shape-customization-tool).
```

Reply:
I have pushed the changes:-- added cornerfamily and removed empty parent.

Reply:
Thanks @bhaktideshmukh! Can you confirm that all of the screens still look correct after this?